Process.c

```diff
@@ -894,6 +895,48 @@ void Process_writeField(const Process* this, RichString* str, ProcessField field
          xSnprintf(buffer, n, "%3ld ", this->priority);
       break;
    case PROCESSOR: xSnprintf(buffer, n, "%3d ", Settings_cpuId(this->settings, this->processor)); break;
+   case SCHEDPOL:
+      if (this->scheduling_policy < 0) {
+         xSnprintf(buffer, n, "N/A   ");
+      } else {
+         #ifdef SCHED_RESET_ON_FORK
+         int pure = this->scheduling_policy & ~SCHED_RESET_ON_FORK;
+         #else
+         int pure = this->scheduling_policy;
+         #endif
+
+         switch (pure) {
+            case SCHED_OTHER:
+               xSnprintf(buffer, n, "OTHER ");
+               break;
+            case SCHED_FIFO:
+               xSnprintf(buffer, n, "FIFO  ");
+               break;
+            case SCHED_RR:
+               xSnprintf(buffer, n, "RR    ");
+               break;
+            #ifdef SCHED_BATCH
+            case SCHED_BATCH:
+               xSnprintf(buffer, n, "BATCH ");
+               break;
+            #endif
+            #ifdef SCHED_BATCH
```

Comment:
Is this `#ifdef SCHED_IDLE` or `#ifdef SCHED_BATCH`? Looks like a typo.

Reply:
Should be `SCHED_IDLE`, good catch!
```suggestion
            #ifdef SCHED_IDLE
```